python/sauron-core/tests/test_classifier.py

```
@@ -0,0 +1,157 @@
+        'correlation_h': -100.0,
+        'correlation_v': 1000.0,
+        'noise': -1000.0,
+        'wavelet_total': 10000.0,
+        'contrast': -10000.0
+    }
+
+    prob = classifier.predict_probability(extreme_metrics)
+    assert isinstance(prob, float)
+    assert 0 <= prob <= 1  # Should still give valid probability
+
+def test_classifier_feature_importance():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_classifier_feature_importance`

Add return type annotation: `None`

(ANN201)